src/app/pages/api/ProductList.jsx

```diff
@@ -1,20 +1,8 @@
-import Products from '../../components/products/Products';
 import useSWR from 'swr';
 
-// const ProductList = ({ products }) => {
-//   return (
-//     <>
-//       <h2>Products bleble</h2>
-//       <Products products={products} />
-//     </>
-//   );
-// };
-
-// export default ProductList;
-
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
-export default function ProductList() {
+const ProductList = () => {
```

Comment:
generalnie to co rozmawialiśmy : to spokojnie mógłby być server side rendering - i ładowanie statycznych postów już po stronie servera : 

https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering